### ISSUE TITLE
How to receive a webhook post from a 3rd party(Tray.io), not the messaging service (Slack)?

### ISSUE THREAD

**Issue by Dakota-Brown** [howdyai/botkit#1898]
### What are you trying to achieve or the steps to reproduce?

TL:DR: How do I receive a webhook post from a third party SAAS solution in my bot and then post that to Slack.

I want to use a third-party SAAS platform (Tray.io) to post a webhook to my Slackbot hosted on Heroku. Once it receives the webhook, I then want to pass the content through and post a message to Slack with an interactive button, that would then, when pressed, open up a modal, accept input, have the user click submit, the payload is then posted back to the Slackbot where that data will then be posted back to Tray.io.

I know how to post a webhook from my Slackbot to Tray. I know how to post to a webhook within Tray. What I don't know how to do using Botkit, is receive that webhook post in the bot and then be able to work with it (i.e. post to Slack).

```
var Botkit = require('botkit');
require('dotenv').config();

if (!process.env.CLIENT_ID || !process.env.CLIENT_SECRET || !process.env.PORT || !process.env.VERIFICATION_TOKEN) {
    console.log('Error: Specify CLIENT_ID, CLIENT_SECRET, VERIFICATION_TOKEN and PORT in environment');
    process.exit(1);
  } else {
    console.log('Good job, you have the variables!')
  }

  var controller = Botkit.slackbot({
    json_file_store: './db_slackbutton_slash_command/',
    debug: true,
    clientSigningSecret: process.env.CLIENT_SIGNING_SECRET,
   })

   controller.configureSlackApp({
    clientId: process.env.CLIENT_ID,
    clientSecret: process.env.CLIENT_SECRET,
    clientSigningSecret: process.env.CLIENT_SIGNING_SECRET,
    scopes: ['commands', 'bot'],
   })

  controller.setupWebserver(process.env.PORT, function(err, webserver){
    controller.createWebhookEndpoints(controller.webserver);
    controller.createOauthEndpoints(controller.webserver, 
      function(err, req, res) {
        if (err) {
          res.status(500).send('ERROR: ' + err);
        } else {
          res.send('Success!');
        }
      });
   });
   
   controller.hears('hi', ['direct_mention', 'mention', 'direct_message'], function(bot, message) {
    bot.reply(message,'Hello <@'+message.user+'>');
   });


   var bot = controller.spawn({
    token: process.env.BOT_TOKEN,
    incoming_webhook: {
      url: 'URL'
    }
  }).startRTM();

  controller.hears('webhook', 'direct_message', function(bot, message) {
    bot.sendWebhook({
      text: "Hey we've got the webhook!"
    },function(err,res) {
      if (err) {
        console.log('web err', err)
      }
    });
  });
```

### What was the result you received?
The code above is my bot currently. I do not know how to receive a webhook post from another SAAS solution and then work with that data.

### What did you expect?
I need help determining how to do this. I have looked for tutorials, combed through Botkits docs, and otherwise explored but cannot seem to put it together.

### Context:

- Botkit version: 0.6.21
- Messaging Platform: Slack
- Node version: 13.1.0
- Os: Mac OS/Heroku
- Any other relevant information:
